examples/dpapi.py

```diff
@@ -568,6 +597,14 @@ def run(self):
     unprotect.add_argument('-entropy', action='store', default=None, required=False, help='String with extra entropy needed for decryption')
     unprotect.add_argument('-entropy-file', action='store', default=None, required=False, help='File with binary entropy contents (overwrites -entropy)')
 
+    # A CREDHIST command
+    credhist = subparsers.add_parser('credhist', help='CREDHIST related functions')
+    credhist.add_argument('-file', action='store', required=True, help='CREDHIST file')
+    credhist.add_argument('-sid', action='store', help='SID of the user')
+    credhist.add_argument('-key', action='store', help='Specific key to use for decryption')
+    credhist.add_argument('-password', action='store', help='User\'s password. If you specified the SID and not the password it will be prompted')
```

Comment:
Modify the help message based on the previous changes
```suggestion
    credhist.add_argument('-password', action='store', help='User\'s password.')
```